src/main/java/seedu/billboard/logic/commands/AddTagCommand.java

```diff
@@ -0,0 +1,89 @@
+package seedu.billboard.logic.commands;
+
+import static seedu.billboard.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.billboard.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.billboard.model.Model.PREDICATE_SHOW_ALL_EXPENSES;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.billboard.commons.core.Messages;
+import seedu.billboard.commons.core.index.Index;
+import seedu.billboard.logic.commands.exceptions.CommandException;
+import seedu.billboard.model.Model;
+import seedu.billboard.model.expense.Expense;
+import seedu.billboard.model.tag.Tag;
+
+/**
+ * Adds tag(s) to an existing expense.
+ */
+public class AddTagCommand extends TagCommand {
+    public static final String COMMAND_WORD = "add";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds tags to the expense identified "
```

Comment:
Missing a `tag` before the command word, the message should probably be like "tag add: Adds tags to the expense identified..."